cognite/client/_api/time_series.py

```diff
@@ -178,29 +175,19 @@ def __call__(
         prep_sort = prepare_filter_sort(sort, TimeSeriesSort)
         self._validate_filter(advanced_filter)
 
-        return self._list_generator(
+        async for item in self._list_generator(
             list_cls=TimeSeriesList,
             resource_cls=TimeSeries,
             method="POST",
             chunk_size=chunk_size,
             filter=filter.dump(camel_case=True),
             advanced_filter=advanced_filter,
             limit=limit,
-            partitions=partitions,
             sort=prep_sort,
-        )
-
-    def __iter__(self) -> Iterator[TimeSeries]:
```

Comment:
Why have the `__iter__` methods been removed? - not mentioned in migration guide.

Reply:
It is mentioned there, below `### Removed`.

Imo it should never have been added, having `__call__` as the way to iterate through data is perfect - and enough. Take instances for example, using `__iter__` can never return anything else than nodes. It also encourages iterating through all available data which we don't want.